src/main/java/sirius/biz/codelists/CodeLists.java

```diff
@@ -49,9 +50,10 @@
  *
  * @param <I> the type of database IDs used by the concrete implementation
  * @param <L> the effective entity type used to represent code lists
+ * @param <T> the effective entity type used to represent translations
  * @param <E> the effective entity type used to represent code list entries
  */
-public abstract class CodeLists<I, L extends BaseEntity<I> & CodeList, E extends BaseEntity<I> & CodeListEntry<I, L>> {
+public abstract class CodeLists<I, L extends BaseEntity<I> & CodeList, T extends BasicTranslations<?>, E extends BaseEntity<I> & CodeListEntry<I, L, T>> {
```

Comment:
do we really need to expose this in the type-system or can't we rely on the interface here?
-> there are tons of references to this type...

Reply:
Also shouldn't there be a getter for translated values anywhere?

Reply:
true, the exposed type is not really necessary and has been removed.

added getter for translated values as well.